src/web/tools/GestureHandlerWebDelegate.ts

```diff
@@ -31,19 +32,16 @@ export class GestureHandlerWebDelegate
       );
     }
 
+    this.ready = true;
+
```

Comment:
Since `updateGestureConfig` is called before `init` method (I'm referring to Gesture Handler class), `delegate` is not initialized when `onEnabledChange` is called for the first time. This results in crashes since we cannot obtain `config`.

Now, there are two ways of solving it:

1. Check if `handler` was initialized - this way we keep everything inside Gesture Handler class and we don't have to worry about delegate not being ready.

2. Check if `delegate` was initialized - I think it is a bit clearer approach as it keeps `web` specific behavior out of Gesture Handler class. `if` statement inside Gesture Handler looks ok and it doesn't mess with Gesture Handler implementation.

Both of those approaches look fine, but I'm more into the second one so I've decided to start with it. Please let me know what do you think @j-piasecki.

Reply:
Yep, the second one sounds good.